continuousflex/protocols/convert.py

```diff
@@ -58,9 +59,10 @@ def modeToRow(mode, row):
     
 def getNMAEnviron():
```

Comment:
This function adds to the LD_LIBRARY_PATH what is installed on the conda environment.
If you face any issue in running Genesis protocols, you may to use it like its used [here](https://github.com/scipion-em/scipion-em-continuousflex/blob/master/continuousflex/protocols/protocol_nma_base.py#L158)